### PR TITLE
Lesson 7

### PR DESCRIPTION
**Pull request by IharDorinel**
made amendments in top section, changed colors to custom properties.
There is a part of my homework without responsive design.

### CODE REVIEW

File: top_section_styles.css [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    --primary-bg: #EAF0F9;
+}
+
+body {
+    margin: 0;
+}
+.big-container {
+    width: 1440px;
+    height: 900px;
+    /*min-height: 100vh;*/
```
Comment (dori17) [inline top_section_styles.css]: don't push comments, please

File: what_do_we_offer.html [hunk 2]
```diff
@@ -2,40 +2,71 @@
...
+    </path>
+    </symbol>
+</svg>
+<p class="heading">What do we offer</p>
 
-    <p class="heading">What do we offer</p>
 
-    <div class="rectangle">
-    </div>
+<!--<div class="big-container">-->
```
Comment (dori17) [inline what_do_we_offer.html]: don't push comments, please

File: what_do_we_offer_styles.css [hunk 3]
```diff
@@ -31,7 +44,76 @@
...
+
+
+
+/*For mobile phones*/
+[class*="col-"] {
+    width: 100%;
+}
+
+/*@media only screen and (min-width: 600px) {*/
+/*    !*For tablets*!*/
```
Comment (dori17) [inline what_do_we_offer_styles.css]: remove pls

File: what_do_we_offer_styles.css [hunk 4]
```diff
@@ -1,28 +1,41 @@
...
+    margin-top: 120px;
 }
 
-.rectangle {
-    margin: auto;
-    width: 100px;
-    height: 4px;
-    background-color: #F5BD41;
-}
+/*.heading:hover:after {*/
```
Comment (dori17) [inline what_do_we_offer_styles.css:23]: this should not be commented. it is required for this homework

**Review by dori17 — CHANGES_REQUESTED** [review]

**Comment by IharDorinel** [comment]
@dori17 I've fixed your comments @romos21

**Pull request merged**